x-pack/plugins/monitoring/public/components/logstash/pipeline_viewer/models/pipeline/if_statement.js

```diff
@@ -29,6 +37,23 @@ export class IfStatement extends Statement {
     this.elseStatements = elseStatements;
   }
 
+  toList(depth, parentId) {
+    const list = [];
+
+    list.push(new IfElement(this, depth, parentId));
+
+    addStatementsToList(list, this.trueStatements, depth, this.id);
+
+    if (this.elseStatements.length) {
+      const elseElement = new ElseElement(this, depth, parentId);
+      list.push(elseElement);
+
+      addStatementsToList(list, this.elseStatements, depth, elseElement.id);
```

Comment:
For consistency I think you should either change the 4th argument in this statement to `this.id` OR change the 4th argument on line 45 above to `ifElement.id` (where `ifElement` would be `new IfElement(...)`). Personally, I'd prefer the latter, given the nature of `this` in JS.

Reply:
The reason for this is that the `ElseElement` class creates a new ID; it's needed for collapsing purposes, since in ConfigView we allow the user to collapse just an else block. I'll do the latter option and define a variable for the `IfElement`.